src/meshes/mesh_io.jl

```diff
@@ -256,7 +280,7 @@ function load_mesh_serial(mesh_file::AbstractString; n_cells_max, RealT)
         end
         mesh = TreeMesh(SerialTree{ndims}, max(n_cells_max, capacity))
         load_mesh!(mesh, mesh_file)
-    elseif mesh_type == "StructuredMesh"
+    elseif (mesh_type == "StructuredMesh") || (mesh_type == "StructuredMeshView")
```

Comment:
```suggestion
    elseif mesh_type in ("StructuredMesh", "StructuredMeshView")
```
I find this easier to read, but that's just a suggestion